Makefile

```diff
@@ -530,7 +534,7 @@ validate.completions: completions/bash/podman
 	. completions/bash/podman
 	if [ -x /bin/zsh ]; then /bin/zsh completions/zsh/_podman; fi
 
-validate: gofmt .gitvalidation validate.completions golangci-lint man-page-check
+validate: gofmt .gitvalidation validate.completions golangci-lint man-page-check codespell
```

Comment:
I'm just concerned about codespell being a bit overzealous and keeping the build.  Will it just ping a warning or stop the submit from going forward?

Reply:
I think we should have this on by default, and see if it causes issues, once I get it into a good state.  I think this would help us keep this clean.

Reply:
Off the bat, this will run first from [the gate task here.](https://github.com/containers/libpod/blob/733c85bdb8c5d57639a59baeb7331c02f2a3e434/.cirrus.yml#L107)  Adding things to the container image is easy, see `contrib/gate/Dockerfile`.  Do that in a separate PR, and once merged, quay will automatically build the image (can be confirmed by logging in and checking it on quay).  Then, re-running the jobs here will start using the new image.

Reply:
I have disabled it by default for now, so we can get this PR merged, and then we can figure out how to get the CI system to gate on it in a future PR.